collect_app/src/main/java/org/odk/collect/android/database/entities/DatabaseEntitiesRepository.kt

```diff
@@ -419,7 +436,8 @@ private class EntitiesDatabaseMigrator :
             """
             CREATE TABLE IF NOT EXISTS ${ListsTable.TABLE_NAME} (
                     $_ID integer PRIMARY KEY, 
-                    ${ListsTable.COLUMN_NAME} text NOT NULL
+                    ${ListsTable.COLUMN_NAME} text NOT NULL,
+                    ${ListsTable.COLUMN_VERSION} text
```

Comment:
> Can this be null?

Yes it would be `null` if you just called `addList` and didn't update the hash. That'd still work fine though as `null` would (or should) be different from any hash resulting from an actual list.

> And shouldn't this by UNIQUE?

No. There's no reason for it to be - we only care about the hash for the list we're processing being different from the current one. Also, two different lists could have exactly the same data resulting in the same hash (or even different data with the same hash in very rare cases).

Reply:
Ok I agree with `UNIQUE` but the case with null shouldn't take place. Am I missing something?

Reply:
We call `#save` (which creates lists) with before `#updateListHash` so it would be `null` between those calls. It'd also be `null` when creating/updating an entity for a list that's not been provided by the server (in `LocalEntityUseCase#updateLocalEntitiesFromForm`) and when just calling `#addList` (although that's only in tests).